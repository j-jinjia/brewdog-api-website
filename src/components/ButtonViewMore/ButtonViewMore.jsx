import React from "react";
import "./ButtonViewMore.scss";
const ButtonViewMore = (props) => {
  const { name, handleViewMore } = props;
  return <button onClick={handleViewMore}>{name}</button>;
};

export default ButtonViewMore;
