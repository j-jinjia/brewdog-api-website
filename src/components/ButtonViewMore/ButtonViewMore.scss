.button{
    background-color: #e66465;
    width:100px;
    height: 40px;
    border:none;
    border-radius: 10px;
    cursor: pointer;
    &:hover{
        opacity: 0.7;
    }
}