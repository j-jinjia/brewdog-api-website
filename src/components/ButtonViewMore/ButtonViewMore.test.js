import { render, screen } from "@testing-library/react";
import userEvent from "@testing-library/user-event";
import ButtonViewMore from "./ButtonViewMore";

it("should match navigation snapshot", ()=>{
    const {container} = render (<ButtonViewMore/>)
    expect(container).toMatchSnapshot();

})

it("should render the button", ()=>{
   render(<ButtonViewMore/>)
   const button = screen.getByRole("button")
   expect(button).toBeInTheDocument();

})