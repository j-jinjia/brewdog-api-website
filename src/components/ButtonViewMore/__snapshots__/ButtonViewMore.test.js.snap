// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match navigation snapshot 1`] = `
<div>
  <button
    class="button"
  />
</div>
`;
